lib/auth/httpfallback.go

```diff
@@ -672,3 +672,60 @@ func (c *Client) SetAuthPreference(ctx context.Context, cap types.AuthPreference
 
 	return nil
 }
+
+// GetClusterAuditConfig gets cluster audit configuration.
+func (c *Client) GetClusterAuditConfig(ctx context.Context, opts ...services.MarshalOption) (types.ClusterAuditConfig, error) {
+	auditConfig, err := c.APIClient.GetClusterAuditConfig(ctx)
+	if err != nil {
+		if !trace.IsNotImplemented(err) {
+			return nil, trace.Wrap(err)
+		}
+	} else {
+		return auditConfig, nil
+	}
+
+	cfg, err := c.GetClusterConfig()
```

Comment:
Not sure whether we care about the `MarshalOption`s anymore but given this data is available we should probably pass it along
```suggestion
	cfg, err := c.GetClusterConfig(opts...)
```
here and below.